test/functional/test-custom-element.js

```diff
@@ -148,26 +153,22 @@ describe('CustomElement', () => {
     }
   }
 
-  const ElementClass = document.registerElement('amp-test', {
-    prototype: createAmpElementProto(window, 'amp-test', TestElement),
-  });
-
-  const StubElementClass = document.registerElement('amp-stub', {
-    prototype: createAmpElementProto(window, 'amp-stub', ElementStub),
-  });
-
-  let sandbox;
-  let resourcesMock;
-  let clock;
-  let testElementGetInsersectionElementLayoutBox;
-  let container;
-
   beforeEach(() => {
-    sandbox = sinon.sandbox.create();
+    win = env.win;
+    doc = win.document;
+    clock = lolex.install(win);
```

Comment:
It's because realWin creates its own window and `sandbox.useFakeClock` only can override global window.